config/database.php

```diff
@@ -117,24 +116,24 @@
         'client' => env('REDIS_CLIENT', 'predis'),
 
         'options' => [
-            'cluster' => env('REDIS_CLUSTER', 'predis'),
+            'cluster' => env('REDIS_CLUSTER', 'redis'),
             'prefix' => env('REDIS_PREFIX', Str::slug(env('APP_NAME', 'laravel'), '_').'_database_'),
         ],
 
         'default' => [
             'url' => env('REDIS_URL'),
-            'host' => env('REDIS_HOST', 'localhost'),
+            'host' => env('REDIS_HOST', '127.0.0.1'),
             'password' => env('REDIS_PASSWORD', null),
-            'port' => env('REDIS_PORT', 6379),
-            'database' => env('REDIS_DB', 0),
+            'port' => env('REDIS_PORT', '6379'),
+            'database' => env('REDIS_DB', '0'),
```

Comment:
Hm curious why these need to be strings?

Reply:
I wasn't sure, but looked into it and [this PR explains it](https://github.com/laravel/laravel/pull/5191)... found the reasoning interesting in the quoted example provided in the description!

Reply:
tldr; Maintains consistency of value type if an env variable for this exists, it'll be a string.

Reply:
oh interesting!